docs/pages/layout.tsx

```diff
@@ -119,7 +119,7 @@ const DocsLayout = ({
   const description = 'Build instant web apps.';
 
   useMetadata({
-    htmlClassName: theme === 'dark' ? 'dark' : undefined,
+    htmlClassName: theme ?? undefined,
```

Comment:
Note: I'm curious about the security implications of setting the classes to whatever the user sets as a cookie. Any thoughts on this are appreciated.